### ISSUE TITLE
Compass trees

### ISSUE THREAD

**Issue by Kingfree25** [DarkPacks/SkyFactory-4#1907]
## Bug Description

 I left the chunk boarder of my island and came back to all the trees leaves textures were compasses.
Normal trees not planted in Bonsai Were fine and not glitched
![image](https://user-images.githubusercontent.com/84342412/118530516-fc428400-b701-11eb-8654-0e5f6227c6da.png)

### Did this behavior use to work in the previous version?
  I'm not sure never played older versions

## Steps to Reproduce (for bugs)

1. Leave chunk board
2. Comeback to Bonsai trees

## Logs
 No crash
* Crash Log:No crash

## World Information
* World Preset: 
* Prestige Enabled: Yes
* Modpack Version world created in: 4.2.2
* Additional Content Installed:No

## Client Information
* Modpack Version: 4.2.2
* Java Version: 8.0_291-b10
* Launcher Used: curse forge
* Memory Allocated: 12 gb
* Server/LAN/Single Player: single player
* Optifine Installed: NO
* Shaders Enabled: No

**Comment by Kingfree25** [comment]
Restarting the world fixies the problem

**Comment by sam-kirby** [comment]
Did you change resource pack when this happened?

**Comment by Kingfree25** [comment]
I did change my video settings before hand but not resource pack sense I just use the defaults. What I fined weirder is that compasses have different tints passed off of the tree. Luckily I haven't gotten the bug yet again but I really don't know what caused it to happen in the first place. I guess it might have been a one time deal.

**Comment by sam-kirby** [comment]
If you adjust certain video settings such as mipmap level then it causes the game to regenerate the texture map. The leaves of the tree are probably just a single texture with different colour tints applied so if the texture map changes and the compass ends up in the slot the leaves used to be in this could happen.

I have been able to recreate this, though it wasn't a compass for me.

As I think we've found the source of the issue and it would likely lead to a performance regression if it was fixed I'm going to close this issue.
